handbook/engineering/security/index.md

```diff
@@ -6,6 +6,14 @@
 
 We think that security is an enabler for the business. Sourcegraph is committed to proactive security, and addressing vulnerabilities in a timely manner. We approach security with a can-do philosophy, and look to achieve product goals while maintaining a positive posture, and increasing our security stance over time.
 
+## Members
+
+- [Dan Mckean](../../../company/team/index.md#dan-mckean-he-him) (Product Manager)
+- FQ2 engineering manager hire, [Bill Creager](../../../company/team/index.md#bill-creager) acting manager until then {#security-eng}
+  - [Elizabeth Stirling](../../../company/team/index.md#elizabeth-stirling-she-her)
+  - [André Eleuterio](../../../company/team/index.md#andré-eleuterio-hehim)
+  - FQ4 security engineer
```

Comment:
Nit - IIRC, our hiring plans have changed. CC @bcreager  - what's the current state after the board meeting?